acceptance/ui/features/internalServices.feature

```diff
@@ -65,47 +68,55 @@ Feature: Internal Services
   Scenario: View details for the Docker Registry service
     When I am on the applications page
       And I view the details for "Internal Services" in the "Applications" table
-      And I view the details for "Docker Registry" in the "Services" table
+      And I view the details for "Docker Registry" in the "Internal Services" table
     Then I should not see an entry for "Elastic Search - Serviced" in the table
       And I should see "Total % Used" in the "CPU Usage" graph
       And I should see "Total bytes" in the "Memory Usage" graph
 
   Scenario: View details for the Elastic Search - LogStash service
     When I am on the applications page
       And I view the details for "Internal Services" in the "Applications" table
-      And I view the details for "Elastic Search - LogStash" in the "Services" table
-    Then I should not see an entry for "Celery" in the table
+      And I view the details for "Elastic Search - LogStash" in the "Internal Services" table
+    Then I should not see an entry for "Kibana" in the table
       And I should see "Total % Used" in the "CPU Usage" graph
       And I should see "Total bytes" in the "Memory Usage" graph
 
   Scenario: View details for the Elastic Search - Serviced service
     When I am on the applications page
       And I view the details for "Internal Services" in the "Applications" table
-      And I view the details for "Elastic Search - Serviced" in the "Services" table
+      And I view the details for "Elastic Search - Serviced" in the "Internal Services" table
     Then I should not see an entry for "Docker Registry" in the table
       And I should see "Total % Used" in the "CPU Usage" graph
       And I should see "Total bytes" in the "Memory Usage" graph
 
+  Scenario: View details for the Kibana service
+    When I am on the applications page
+    And I view the details for "Internal Services" in the "Applications" table
+    And I view the details for "Kibana" in the "Internal Services" table
+    Then I should not see an entry for "Zookeeper" in the table
+    And I should see "Total % Used" in the "CPU Usage" graph
+    And I should see "Total bytes" in the "Memory Usage" graph
+
```

Comment:
I know the indentation is purely decoration (tight rhymes), but this doesnt match the style of all the other stuff, and may benefit from being changed.